### PR TITLE
changed banner divider css #6713

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

**Comment by GauravButola** [comment]
LGTM

File: theme/templates/crm.html [hunk 1]
```diff
@@ -6,16 +6,9 @@
...
     <div class="text-center section-space">
-      <h1 class="white-text-color">{% trans %}Customer relationship management{% endtrans %}
-      </h1>
-      <div class="col-sm-12 col-xs-12" >
-        <div class="col-sm-2 col-xs-4 col-xs-offset-4 col-sm-offset-5 app-banner-divider">
-        </div>
-      </div>
-      <div>
+      <h1 class="white-text-color app-banner-heading heading">{% trans %}Customer relationship management{% endtrans %}</h1>
         <p class="white-text-color">{% trans %}Tryton’s CRM is designed to improve lead generation and keep your customer relationships on track. The minimalistic CRM application tracks contacts, activities and opportunities.{% endtrans %}
```
Comment (tarunbhardwaj) [inline theme/templates/crm.html]: de indent

File: theme/templates/ecommerce.html [hunk 2]
```diff
@@ -6,16 +6,9 @@
...
     <div class="text-center section-space">
-      <h1 class="white-text-color">{% trans %}e-Commerce{% endtrans %}
-      </h1>
-      <div class="col-sm-12 col-xs-12" >
-        <div class="col-sm-2 col-xs-4 col-xs-offset-4 col-sm-offset-5 app-banner-divider">
-        </div>
-      </div>
-      <div>
+      <h1 class="white-text-color app-banner-heading heading">{% trans %}e-Commerce{% endtrans %}</h1>
         <p class="white-text-color">{% trans %}Tryton ecommerce app easily adapts and scales to the unique selling and buying scenarios of your specific industry needs. It deliver consistent and engaging shopping experiences on any device or anywhere your customers shop.{% endtrans %}
```
Comment (tarunbhardwaj) [inline theme/templates/ecommerce.html]: deindent

File: theme/templates/foundation.html [hunk 3]
```diff
@@ -7,15 +7,9 @@
...
   <div class="container">
     <div class="text-center section-space">
-      <h1 class="white-text-color">{% trans %}Collaborate freely{% endtrans %}</h1>
-      <div class="col-sm-12 col-xs-12" >
-        <div class="col-sm-2 col-xs-4 col-xs-offset-4 col-sm-offset-5 app-banner-divider">
-        </div>
-      </div>
-      <div>
+      <h1 class="white-text-color app-banner-heading heading">{% trans %}Collaborate freely{% endtrans %}</h1>
         <p class="white-text-color">{% trans %}We strive to make Tryton, an Enterprise open source project where people collaborate with open source and open mind.{% endtrans %}
```
Comment (tarunbhardwaj) [inline theme/templates/foundation.html]: deindent

File: theme/templates/tag.html [hunk 4]
```diff
@@ -2,7 +2,7 @@
 {% extends 'index.html' %}
 {% block title %} {{ tag }} | {{ super() }} {% endblock title %}
 {% block heading %}
-  <h1 class="white-text-color heading">Blogs tagged '{{ tag }}'</h1>
+  <h1 class="white-text-color app-banner-heading heading">Blogs tagged '{{ tag }}'</h1>
```
Comment (tarunbhardwaj) [inline theme/templates/tag.html]: trans ?

**Pull request merged**